### DISCUSSION TITLE
Process Auto Start

### DISCUSSION THREAD

**Discussion by MohamedAgiez** [flowsforapex/apex-flowsforapex#708] [Q&A]
Is it Possible to auto start Process ? , I tried Scheduling section but this is not working for me.
![Scheduling](https://github.com/flowsforapex/apex-flowsforapex/assets/8095868/8fe50047-d0d6-48d8-8e49-392b02a0246e)

**Comment by rallen2010** [comment]
Hi.  The Scheduling section of the Process definition is for you to specify the required end date of the process - when the process is due to be completed.   This can be calculated at process start time using the various expression types, and is then available for display throughout the process execution time (and afterwards).  It can be updated while the process is running.
To start a process instance at a given time, you have two choices:

1. Using BPMN Timer Start Event.   Rather than using a standard (null) Start Event, you could use a Timer Start Event.    The process start will then await until the required time.    Use this if you want to start one process, but have the start wait until a later time.  See https://flowsforapex.org/latest/timer-events/#timer-start-event for documentation.
2. Using APEX Automations.  Automations can be used to start one, or a series of many, process instances based on time or a database query result.  You can use the Automation Action to specify that you want the automation to create and start a Flows for APEX process instance.  You can either use the PL/SQL API (flows_api_pkg.create_process, start_process, etc. ) to start an instance in your code, or, starting in Flows for APEX v 23.1, you can do this declaratively as the **Flows for APEX - Manage Instance Plugin** has been enabled for automations.   Use this if you want to process to start multiple times (for example, start a new process instance every day at 1 AM to run a daily process, or from a query for each record returned by the query - for example, re-order items that are out of stock automatically, with a separate process for each item  needing to be ordered).  

Hope this helps.  I will make a note to document this better in the product documentation....

**Reply by MohamedAgiez** [reply]
Thanks Rallen for your support, I will try both options and feedback soonest.
